src/commandDetails/coin/leaderboard.ts

```diff
@@ -0,0 +1,72 @@
+import { container, SapphireClient } from '@sapphire/framework';
+import { Client, MessageEmbed } from 'discord.js';
+import {
+  CodeyCommandDetails,
+  CodeyCommandResponseType,
+  getUserIdFromMessage,
+  SapphireMessageExecuteType,
+  SapphireMessageResponse
+} from '../../codeyCommand';
+import { getCurrentCoinLeaderboard, getCoinBalanceByUserId, UserCoinEntry } from '../../components/coin';
+import { EMBED_COLOUR } from '../../utils/embeds';
+
+const getCurrentCoinLeaderboardEmbed = async (
+  client: SapphireClient<boolean>,
+  leaderboard: UserCoinEntry[],
+  currentUserId: string,
+  limit = 10
+): Promise<MessageEmbed> => {
+  const currentLeaderboardEmbed = new MessageEmbed().setColor(EMBED_COLOUR).setTitle('Current CodeyCoin Leaderboard');
```

Comment:
I think "Current" can be removed from the title, so it would just be "CodeyCoin Leaderboard" for conciseness.

Reply:
I've implemented this change